### PR TITLE
Missing syscall in krfctl profile do not fail

### LINKED ISSUE #68: Missing syscall might abort krfctl -P
When there is a missing syscall, a given `krfctl -P <profile>` might `abort`:
```
root@ubuntu-bionic:/home/vagrant/krf# krfctl -P fs
krfctl: couldn't find syscall: sysctl
```

From @woodruffw :
> I guess `fault_syscall` could return a `bool` and we could push the error behavior one layer up

Related code: https://github.com/trailofbits/krf/blob/master/src/krfctl/linux/linux.c#L46

### PR DESCRIPTION
**Pull request by layderv**
closes #68

### CODE REVIEW

File: src/krfctl/freebsd/freebsd.c [hunk 1]
```diff
@@ -26,12 +26,13 @@
...
 #define TARGETING_NAME KRF_PROC_DIR "." KRF_TARGETING_FILENAME
 
-void fault_syscall(const char *sys_name) {
+int fault_syscall(const char *sys_name) {
   const char *sys_num;
   unsigned int syscall;
 
   if (!(sys_num = lookup_syscall_number(sys_name))) {
-    errx(1, "couldn't find syscall %s", sys_name);
+    fprintf(stderr, "WARNING: couldn't find syscall %s", sys_name);
```
Comment (woodruffw) [inline src/krfctl/freebsd/freebsd.c:34]: Use `warnx` here, for consistency with `err`/`errx` use 😉

File: src/krfctl/linux/linux.c [hunk 2]
```diff
@@ -44,7 +44,8 @@ void fault_syscall(const char *sys_name) {
   }
 
   if (!(sys_num = lookup_syscall_number(sys_name))) {
-    errx(1, "couldn't find syscall: %s", sys_name);
+    fprintf(stderr, "WARNING: couldn't find syscall: %s", sys_name);
```
Comment (woodruffw) [inline src/krfctl/linux/linux.c:47]: Same here.

**Comment by woodruffw** [comment]
Thanks for doing this! I added some nits.

File: src/krfctl/freebsd/freebsd.c (hunk 1 above)
Comment (layderv) [reply src/krfctl/freebsd/freebsd.c]: Updated. Thanks

**Comment by layderv** [comment]
@woodruffw updated, let me know if it's better now

**Comment by woodruffw** [comment]
@layderv Looks great, thank you! Will merge in a moment.

**Review by woodruffw — APPROVED** [review]

**Pull request merged**